AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -489,10 +487,8 @@ - (NSURL *)assetURL
 
 - (void)setAsset:(AVAsset *)asset
 {
-  ASDN::MutexLocker l(__instanceLock__);
-  
   if (ASAssetIsEqual(asset, _asset) == NO) {
```

Comment:
should this be self.asset instead of _asset? Or do we need to lock around the check?